src/bin/python.rs

```diff
@@ -185,9 +185,7 @@ impl Argv0Program {
     fn to_command(&self) -> Command {
         let mut args = env::args_os();
         let mut cmd = Command::new(self.argv0());
-        if let Some(arg0) = args.next() {
-            cmd.arg0(arg0);
```

Comment:
The arg0 make `python -c 'import sys; print(sys.prefix)'` not equal `python --prefix`